package.json

```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "babel-preset-es2015": "^6.24.1",
     "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
```

Comment:
Why did it need stage-2 here? I don't see anything here using it, or am I missing something obvious here.